### PR TITLE
Update minimum-requirements.md

### PR DESCRIPTION
**Pull request by msbemba**
updated M365 business premium.reference https://docs.microsoft.com/en-us/microsoft-365/admin/misc/microsoft-365-business-faqs?view=o365-worldwide#can-microsoft-365-business-premium-customers-use-microsoft-defender-for-endpoint

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/6811

File: microsoft-365/security/defender-endpoint/minimum-requirements.md [hunk 1]
```diff
@@ -44,6 +44,7 @@ Microsoft Defender for Endpoint requires one of the following Microsoft volume l
 - Windows 10 Enterprise E5
 - Windows 10 Education A5
 - Microsoft 365 E5 (M365 E5) which includes Windows 10 Enterprise E5 or Windows 11 Enterprise E5
+- Microsoft 365 Business Premium When purchased from  [Microsoft Cloud Solution Provider](https://partner.microsoft.com/membership/cloud-solution-provider/find-a-provider)
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/minimum-requirements.md]:
```suggestion
- Microsoft 365 Business Premium when purchased from a [Microsoft Cloud Solution Provider](https://partner.microsoft.com/membership/cloud-solution-provider/find-a-provider)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@mjcaparas Please merge this PR. Thanks

**Pull request merged**